rdkit/Chem/Scaffolds/MurckoScaffold.py

```diff
@@ -91,7 +91,7 @@ def _pyGetScaffoldForMol(mol):
       mol = Chem.DeleteSubstructs(mol, patt)
   for atom in mol.GetAtoms():
     if atom.GetAtomicNum() == 6:
-      if atom.GetNoImplicit() and atom.GetExplicitValence() < 4:
+      if atom.GetNoImplicit() and atom.GetValence(getExplicit=True) < 4:
```

Comment:
In this situation isn't this the same as `GetTotalValence()`?

Reply:
yes, but this is more explicit (sorry, unavoidable) and we avoid the unneccessary call to GetValence(IMPLICIT)